meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -63,6 +122,9 @@ OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
 # Explicit machine declaration by build job:
 # MACHINE = "$TARGET_MACHINE"
 
+# Build number shared by all jobs across the different machines:
+# BUILD_ID = "<parent-build-timestamp (of Jenkins top job)>" + "-build-" + "<jenkins-top-job-build-number>"
+
```

Comment:
BB_NUMBER_PARSE_THREADS is a bitbake variable. It is written in config file.
BUILD_ID is not a bitbake or any recipe variable. it is not written in config during build time.

That's difference. You don't care, I do care about what and how it is used, to eliminate surprises in future.